src/util/common-utils.ts

```diff
@@ -134,6 +135,35 @@ function installApp(
     });
 }
 
+async function reinstallApp(params: {
+  flags: FlagInput;
+  type: string;
+  orgUid: string;
+  manifestUid: string;
+  configType: ConfigType;
+  developerHubBaseUrl: string;
+}): Promise<void> {
+  const { type, orgUid, manifestUid, developerHubBaseUrl, flags } = params;
+
+  const payload = {
+    target_type: type,
+    target_uid: (flags["stack-api-key"] as any) || orgUid,
+  };
+
+  const url = `https://${developerHubBaseUrl}/manifests/${manifestUid}/reinstall`;
+  try {
+    const result = await apiRequestHandler({
+      orgUid,
+      queryParams: params,
```

Comment:
@harshithad0703 , What is the purpose of sending entire parameters as queryParams?Handle it properly